doc/en/usage.rst

```diff
@@ -189,6 +189,187 @@ in your code and pytest automatically disables its output capture for that test:
   for test output occurring after you exit the interactive PDB_ tracing session
   and continue with the regular test run.
 
+.. _logging:
+
```

Comment:
If a user has ``pytest-catchlog`` installed what will happen? Probably they will step in each other's toes.

Perhaps we should put a note here:

```rst
... note::
    This feature is a drop-in replacement for the `pytest-catchlog <https://pypi.org/project/pytest-catchlog/>`_ plugin
    and they will conflict with each other. The backward compatibility API with ``pytest-capturelog`` has been dropped 
    when this feature was introduced, so if for that reason you still need ``pytest-catchlog`` you can disable the internal
    feature by adding to your ``pytest.ini``:

    .. code-block:: ini

        [pytest]
        addopts=-p no:logging
```